network/src/peers/peers.rs

```diff
@@ -93,10 +93,10 @@ impl<S: Storage + Send + Sync + 'static> Node<S> {
             }
         }
 
-        // Attempt to connect to the default bootnodes of the network if the node has no active
+        // Attempt to connect to the default bootnodes and initial peers of the network if the node has no active
```

Comment:
```suggestion
        // Attempt to connect to the default bootnodes of the network if the node has no active
```
this is now obsolete, no?